### PR TITLE
Test conda build 2

### PR DESCRIPTION
**Pull request by ocefpaf**
@jochym I am testing the full `numpy` build matrix here with `conda-build 2`. Fingers crossed!

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,31 +12,29 @@ source:
...
+    - numpy x.x
     - setuptools
-    - toolchain
   run:
     - python
-    - numpy 1.11.*
+    - numpy x.x
     - ase
     - scipy
-    - yaml
```
Comment (ocefpaf) [inline recipe/meta.yaml]: It looks like the code only uses `pyyaml`. Please correct me if I am wrong.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,31 +12,29 @@ source:
...
     - pyyaml
-    - future
```
Comment (ocefpaf) [inline recipe/meta.yaml]: It does not look like the code imports this.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,31 +12,29 @@ source:
...
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
-    - numpy 1.11.*
+    - numpy x.x
     - setuptools
-    - toolchain
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This is probably not needed here. The tests will confirm that. So let's remove more variables that can interfere with the build.

**Comment by ocefpaf** [comment]
Alright! Everything works! let's get all `numpy` manifestations for `phonopy` out there.

File: recipe/meta.yaml (hunk 1 above)
Comment (jochym) [inline recipe/meta.yaml]: I think so. Not any more. It used to.

File: recipe/meta.yaml (hunk 2 above)
Comment (jochym) [inline recipe/meta.yaml]: Not any more as well

**Comment by jochym** [comment]
Nice. I was too busy today to do it myself. Thanks @ocefpaf ! 
Anything left for me to do?

**Comment by ocefpaf** [comment]
> Anything left for me to do?

You already run all the tests in the recipe but it is always nice to get field confirmation that this works :grimacing: 

Let me know if you find any issues with the extra `numpy` versions out there.

**Pull request merged**